router.go

```diff
@@ -25,16 +24,13 @@ type RouterOption func(Router) Router
 // The following router names are accepted:
 // * `gorilla` which uses github.com/gorilla/mux (on by default)
 // * `stdlib` to utilize the standard library's http.ServeMux
-// * `httprouter` which uses github.com/julienschmidt/httprouter
 //
 // If the supplied name does not match any known router, `gorilla` will be used.
 // If a user wishes to supply their own router implementation, the `CustomRouter` option
 // is available.
 func RouterSelect(name string) RouterOption {
 	return func(_ Router) Router {
 		switch name {
-		case "httprouter":
-			return &FastRouter{httprouter.New()}
```

Comment:
Do this cold turkey or give users a warning that this has been deprecated and now uses GorillaRouter?

Reply:
We dont have a context at this point so any log message would get lost into nowhere land :(

I think we're fine cold turkey, though. I've been surveying users and no one has piped up in defense of httprouter.

If all else fails, the repo is already tagged so those poor quite souls could pin the repo until they move to the "slower" router impls.

Reply:
Ah, good point, you don't have `ctx` here yet.

🚢 it